lib/chef/resource/homebrew_tap.rb

```diff
@@ -74,7 +74,11 @@ class HomebrewTap < Chef::Resource
       #
       # @return [Boolean]
       def tapped?(name)
-        base_path = ["#{::File.dirname(which("brew"))}/../homebrew", "#{::File.dirname(which("brew"))}/../Homebrew", "/opt/homebrew", "/usr/local/Homebrew", "/home/linuxbrew/.linuxbrew"].uniq.select { |x| Dir.exist?(x) }.first
+        base_path = ["#{::File.dirname(which("brew"))}/../homebrew", "#{::File.dirname(which("brew"))}/../Homebrew", "/opt/homebrew", "/usr/local/Homebrew", "/home/linuxbrew/.linuxbrew"].uniq.select do |x|
+                      next if x == false
+
+                      Dir.exist?(x)
+                    end.first
```

Comment:
@Stromweld check the alignment here and also we'll want to have a chef-18 backport.